goodeggs-deploy.sh

```diff
@@ -49,6 +49,13 @@ if [ -f "$statsfile" ]; then
   # For example, apps are known to log "STATUS_API_TOKEN required, but not set. Configuring goodeggs-status in simulate mode"
   # within a full JSON log line with a timestamp - which makes this hashing nondeterministic. :(
   statsfile_hash=$(babel-node -e "console.log(JSON.stringify(require('./$statsfile')))" | tail -1 | md5sum | cut -d ' ' -f 1)
+  # Assumes jq is installed. Currently provided by ecru and by our Buildkite image.
+  goodeggs_stats_cli_version=$(jq ".devDependencies[\"goodeggs-stats-cli\"]" package.json)
```

Comment:
it should be available everywhere, but why not just use `node`?:

```suggestion
  goodeggs_stats_cli_version=$(node -e 'console.log(require("./package.json").devDependencies["goodeggs-stats-cli"])')
```

Reply:
makes sense, good call 👍🏼